Lib/fontTools/feaLib/parser.py

```diff
@@ -787,9 +792,11 @@ def parse_table_(self):
 
     def parse_table_GDEF_(self, table):
         statements = table.statements
-        while self.next_token_ != "}":
-            self.advance_lexer_()
-            if self.is_cur_keyword_("Attach"):
+        while self.next_token_ != "}" or (not self.ignore_comments and len(self.cur_comments_)):
```

Comment:
Actually, does the parser really need to have a special mode for ignoring comments? If it always recognized comments, and always added them to the syntax tree, the API would be simpler. One less option to worry about.